synapse/lib/layer.py

```diff
@@ -3629,6 +3649,9 @@ def _editTagPropSet(self, buid, form, edit, sode, meta):
         sode['tagprops'][tag][prop] = (valu, stortype)
         self.setSodeDirty(buid, sode, form)
 
+        formabrv = self.setPropAbrv(form, None)
```

Comment:
Do we need to add this (and edits below) if the sode already exists?

Reply:
Updated this to only set the abrv/byform if the sode is new.